internal/clioptions/clioptions.go

```diff
@@ -71,6 +71,8 @@ type CLIOptions struct {
 	MarketplaceItemVersion string
 	// MarketplaceItemObjectID is the _id of a Marketplace item
 	MarketplaceItemObjectID string
+	// Fetch only public items
+	Public bool
```

Comment:
```suggestion
	MarketplaceFetchOnlyPublic bool
```
This struct is not domain-specific, isn't it better to have prefixes to avoid confusion?

Reply:
yep, I agree, but it's not a `FetchOnly`, going to use a more domain specific name

Reply:
Ok, in this case you have to fix the comment above too, it still says fetch only